.gitignore

```diff
@@ -3,6 +3,7 @@
 .token.json
 konnector-dev-config.json*
 fixtures/*
+data/*
```

Comment:
must be added to konnectors/cozy-konnector-template

Reply:
Right, this is done but pushed yet

Reply:
And this needs the release of next cozy-konnector-libs